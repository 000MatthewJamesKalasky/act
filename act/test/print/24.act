bool a, b, c;

a = b;
b = c;
