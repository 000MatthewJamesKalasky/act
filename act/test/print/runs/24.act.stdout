

/* instances */
bool b;
bool a;
bool c;

/* connections */
a=c;a=b;
